core/domain/topic_domain.py

```diff
@@ -45,7 +45,7 @@
 # compatibility with previous change dicts.
 TOPIC_PROPERTY_NAME = 'name'
 TOPIC_PROPERTY_ABBREVIATED_NAME = 'abbreviated_name'
-TOPIC_PROPERTY_THUMBNAIL_FILENAME = 'thumbnail_filename'
+TOPIC_PROPERTY_THUMBNAIL_FILENAME_AND_SIZE_IN_BYTES = 'thumbnail_filename'
```

Comment:
Keep the name as-is, the size should be changed transparently

Reply:
Done.